javaauth/test/marathon_auth_plugin_test.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import unittest
+import os
+import socket
+import argparse
+import json
+import sys
+import httplib
+import urllib
+import requests
+import base64
+import subprocess
+import time
+
+def post_data(url,headers,payload):
+    result = requests.post(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def get_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.get(url,data=payload,headers=headers)
+    return result.status_code
+
+def put_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.put(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def delete_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.delete(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def store_original_remote_file():
+    try:
+        user = os.getlogin()
+        remote_path = "/data/marathon/plugins/user-permissions.json"
+        hostname1 = user+"@daldevmesoszk01"
```

Comment:
I believe what you are doing is - saving the existing `user-permissions.json` file before you SCP your test `permissions.json` which is correct thing to do, just use the hostname supplied on the command line, This same tests will also be ran on other environment as well.